backend/src/main/java/com/htecgroup/skynest/model/request/PermissionGrantRequest.java

```diff
@@ -0,0 +1,24 @@
+package com.htecgroup.skynest.model.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class PermissionGrantRequest {
+
+  @NotNull(message = "user id cannot be null or empty")
+  private UUID grantedTo;
+
+  @NotNull(message = "bucket id cannot be null or empty")
+  private UUID bucketId;
```

Comment:
Good enought for this PR, but later we can maybe add a generic way for granting permissions.

Reply:
this really should be objectId, for future (re)use

Reply:
it's changed